### ISSUE TITLE
Atom/RSS feed

### ISSUE THREAD

**Issue by bronikowski** [czep/czep.net-gatsby#1]
I can't believe I'm opening a ticket against a blog… but here we are.

I had your blog in my tiny, curated list of things I want to read in my down time. I normally don't pay attention to a rss2email log but few days ago I noticed that:

```2020-02-20 16:27:29,252 [ERROR] HTTP status 404 fetching feed czep ```

I went to check it out and saw you moved your generator. I'd love to have your feed back!

**Comment by czep** [comment]
I'm glad you raised this, thank you for bringing this to my attention!  I had completely overlooked the RSS feed.  It should be an easy fix to add this.

**Comment by czep** [comment]
Fixed in 3a1f515582b00a5af23c665af5e77c92efe47e02

**Comment by bronikowski** [comment]
Just confirming it:
```
λ ~/ r2e.sh list | grep czep
26: [*] czep (https://czep.net/feed.xml -> emil@mydomain)
λ ~/ r2e.sh run 26
λ ~/ echo $?
0
```
